pkg/relayer/proxy/errors.go

```diff
@@ -15,4 +15,5 @@ var (
 	ErrRelayerProxyInvalidRelayResponse      = sdkerrors.Register(codespace, 7, "invalid relay response")
 	ErrRelayerProxyServiceEndpointNotHandled = sdkerrors.Register(codespace, 8, "service endpoint not handled by relayer proxy")
 	ErrRelayerProxyUnsupportedTransportType  = sdkerrors.Register(codespace, 9, "unsupported proxy transport type")
+	ErrRelayerProxyInternalError             = sdkerrors.Register(codespace, 10, "internal error")
```

Comment:
Optional: Is there an opportunity to create a custom more self desriptive error wherever this  is used?

Ignore if not applicable.

Reply:
Calls to this error systematically wrap the original error. This one is used to identify internal errors (using `errors.Is`) by the `replyWithError` function so it knows that it shouldn't send the original one.